debian/pop-default-settings.postinst

```diff
@@ -84,12 +93,11 @@ case "$1" in
         done
 
         # Add Pop proprietary repo if it is missing.
-        RELEASE="$(lsb_release -cs)"
-        REPO="deb http://apt.pop-os.org/proprietary ${RELEASE} main"
-        if ! grep "${REPO}" /etc/apt/sources.list > /dev/null
+        shopt -s nullglob
+        if ! grep "${REPO}" /etc/apt/sources.list{,.d/*.list,.d/*.source}
```

Comment:
I think we should be checking for `.d/*.sources` instead of `.d/*.source`. Right now, if I change `Enabled: yes` to `Enabled: no`, the check is failing and the repo is being re-enabled every time the package is reconfigured. (`.sources` is the correct extension according to the apt [documentation](https://manpages.debian.org/stretch/apt/sources.list.5.en.html).)

Reply:
Good catch

Reply:
Thanks for catching that, fixed!